### DISCUSSION TITLE
QT/ECG chart subteam discussion

### DISCUSSION THREAD

**Discussion by xni7** [SafetyGraphics/safetyGraphics#659] [Chart Discussion]
@mli1 , let's use this thread to discuss QT related topics. Below are the links to historical discussions

•	QT Pull request and code evolution  https://github.com/SafetyGraphics/safetyCharts/pull/90
•	Github discussion on QT code development: https://github.com/SafetyGraphics/safetyGraphics/discussions/644#discussioncomment-1508597

**Comment by mli1** [comment]
@xni7  for the outlier plot, can you add a bit more labels to the dotted line? It seems to be not very clear to the reader what the diagonal line stands for. Can you add 450msec to the line? also, can we add another two dotted diagonal lines representing 480msec and 500msec?
